scrapper.py

```diff
@@ -25,17 +38,38 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
+
+    def __init__(self, seed_urls, total_max_articles):
+        self.seed_urls = seed_urls
+        self.total_max_articles = total_max_articles
+        # list with urls to articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        all_urls_bs = article_bs.find_all('div', class_='news-item-title')
+        urls_to_articles = []
+        for url_bs in all_urls_bs:
+            full_url = url_bs.find('a')['href']
+            urls_to_articles.append(full_url)
+
+        for url_to_article in urls_to_articles:
+            if len(self.urls) < self.total_max_articles:
+                self.urls.append(url_to_article)
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed_url in self.seed_urls:
+            sleep(random.randint(1, 5))
+            response = requests.get(url=seed_url, headers={
+                'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36'
+                            '(KHTML, like Gecko) Chrome/98.0.4758.109 Safari/537.36'
+            })
```

Comment:
headers are constant, so it is better to import them from `constants.py` like you do with http_pattern